web-console/knikubevirt/Create-vm/step-5-review/wizard-review.md

```diff
@@ -0,0 +1,36 @@
+# Create Virtual Machine Wizard - Review Step
+
+The purpose of this doc is to update the current create VM designs to align with OpenShift and Patternfly 4.
+
+Some things to note:
+
+The “Create virtual machine” action has been updated to “Review and create”.
+Once the user has clicked this button they will taken to the Review screen to verify their settings.
+
+![user clicks review and create button](img/review-and-create-action.png)
+
+The action “Create virtual machine” has been moved to the footer with the other actions.
+This encourages the user to check their settings quickly before creating the VM.
+
+![user checks settings before creating vm](img/check-settings.png)
+
+When the user clicks “Create virtual machine” they will then be taken to the list view where they can see the VM being started.
+This replaces the step where the modal is left opened and requires more clicks from the user. Leaving the modal open only allows them to watch the process in the list view, but they could not cancel it or go back.
+
+![vm is being started](img/vm-starting.png)
+
+If there is an error with the VM starting up they would see this in the list view with the option to look at the events or try again or go and edit the VM’s setup.
```

Comment:
It would be good to go over this with @jelkosz to confirm each of these errors and ensure we can provide these capabilities. 
We have to ensure that both in the wizard flow and in the list view we can provide a similar experience. Would editing the VM details always be through the overview or can they make changes in the wizard flow as well by going back?
I think it would be helpful to @jelkosz for us to think about this and propose a design for it. WDYT?